File: src/main/scala-graphql/io/github/graphql/model/AddReactionMutationResponse.scala
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
class AddReactionMutationResponse extends GraphQLResult[JMap[String, AddReactionPayloadTO]] {

    def addReaction(): AddReactionPayloadTO = {
        val data: JMap[String, AddReactionPayloadTO] = getData
        if (data != null) data.get(AddReactionMutationResponse.OPERATION_NAME) else null
    }

}

object AddReactionMutationResponse {

    private final val OPERATION_NAME: String = "addReaction"

}

File: src/main/kotlin-graphql/io/github/graphql/kotlin/model/AddReactionMutationResponse.kt
package io.github.graphql.kotlin.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult

@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-07T19:48:14+0800"
)
open class AddReactionMutationResponse : GraphQLResult<MutableMap<String, AddReactionPayloadTO?>>() {

    companion object {
        const val OPERATION_NAME: String = "addReaction"
    }

    fun addReaction(): AddReactionPayloadTO? {
        val data: MutableMap<String, AddReactionPayloadTO?> = super.getData()
        return data[OPERATION_NAME]
    }
}
